python/src/com/jetbrains/python/psi/impl/stubs/PyClassStubImpl.java

```diff
@@ -48,17 +48,34 @@
   @Nullable
   private final String myDocString;
 
+  @Nullable
+  private final List<String> mySuperClassesText;
+
   public PyClassStubImpl(@Nullable String name,
                          @Nullable StubElement parentStub,
                          @NotNull Map<QualifiedName, QualifiedName> superClasses,
-                         @NotNull List<String> superClassesText,
+                         @NotNull List<String> subscriptedSuperClassesText,
                          @Nullable QualifiedName metaClass,
                          @Nullable List<String> slots,
                          @Nullable String docString,
                          @NotNull IStubElementType stubElementType) {
+    this(name, parentStub, superClasses, subscriptedSuperClassesText, null, metaClass, slots, docString, stubElementType);
+  }
+
+  public PyClassStubImpl(
+    @Nullable String name,
```

Comment:
Could you please change the formatting so that the indent of the parameter list doesn't clash with that of the function body (unfortunately, there is no unambigous indent guidelines for Java as in PEP 8).

Reply:
@east825 That’s true but there are IDEA formatter settings for the intellij-community project. Did I forget to format when checking in or is that formatting option not supported?